CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2022-08-12
+### Added
+* [ListObjects API](https://openfga.dev/api/service#/Relationship%20Queries/ListObjects)
+
+  The ListObjects API provides a way to list all of the objects (of a particular type) that a user has a relationship with. It provides a solution to the [Search with Permissions (Option 3)](https://openfga.dev/docs/interacting/search-with-permissions#option-3-build-a-list-of-ids-then-search) use case for access-aware filtering on smaller object collections.
+
```

Comment:
```suggestion
Note that that the current implementation has very high latency, and adds some configuration options to limit the response.

Warning: If the configuration options to limit the response are passed, the caller will only get partial data - pagination is not possible on this endpoint.

This change implements the [ListObjects RFC](https://github.com/openfga/rfcs/blob/main/20220714-listObjects-api.md)
```

Reply:
[TODO: Link to PR introducing the config options]
[TODO: Add a sample call like the one below]
```shell
$ curl -X POST ${FGA_SERVER_URI}/stores/${FGA_STORE_ID}/list-objects \
  -H 'Content-Type: application/json'
  -d '{"user":"anne","type":"document","relation":"can_view"}'

$ {"object_ids":["object_id_1","object_id_2"]}
```

Reply:
I've update the section based on this guidance. Let me know what you think about the new changes.

Reply:
> [TODO: Add a sample call like the one below]

@rhamzeh how do you feel about limiting sample API calls and other info like that to the openfga.dev documentation site?

Reply:
I personally think CHangelogs should include just enough context for a developer to be able to pick it up without having to go to the website, especially for:
- breaking changes
- bigger features such as this one

Some of the reasons why I like to include code snippets:
- I can easily grok what the API looks like
- If a change happens in the future, but a developer is using an earlier version, they do not have to wonder what the API looked like at that point in time, they can just quickly skim the changelog to find out

And in general I haven't heard people ever complain about having too much info in the Changelog :)